content/integrations/pingdom.md

```diff
@@ -23,7 +23,7 @@ At this time we track the response_time metric for any sites you configure on th
     (If you have a Team account, you can use your own credentials and specify the account you wish to pull checks from.)
 1.  You can ignore some checks by unchecking them or add some tags to the events that are going to be generated.
 
-Note: Pingdom does not provide an API for transaction checks, so we’re not able to show them in Datadog. 
+Note: Pingdom does not provide an API for transaction checks, so we’re not able to show them in Datadog.
```

Comment:
I think this is now available @technovangelist:  https://www.pingdom.com/resources/api#ResourceProbes